proc_CPMG.py

```diff
@@ -53,10 +53,12 @@
 'tE','#').set_units('tE','scan #'))
     s = s['t2':f_range]
     s = s.C.sum('t2')
+    # {{{ (FROM REVIEW) what is going on here???
     if spincore:
         CPMG=s
     else:
         CPMG = s['indirect',-1]
+    # }}}
```

Comment:
what's going on here?

Reply:
so for my bruker data for some reason there still exists a dimension ['indirect',1]. Now this might be a result of us futzing around trying to find a good pp and maybe I am processing bad bruker data... When I do this however (CPMG=s['indirect',-1]) it does produce a okay decay curve and is able to fit.

Reply:
yes, but then the solution is
`if 'indirect' in s.dimlabels and ndshape(s)['indirect']==1: ....`